medcat/meta_cat.py

```diff
@@ -77,16 +77,31 @@ def get_model(self, embeddings: Optional[Tensor]) -> nn.Module:
                 The embedding densor
 
         Raises:
-            ValueError: If the meta model is not LSTM
+            ValueError: If the meta model is not LSTM or BERT
 
         Returns:
             nn.Module:
                 The module
         """
         config = self.config
+        from medcat.utils.meta_cat.models import LSTM
+        from medcat.utils.meta_cat.models import BertForMetaAnnotation
```

Comment:
Similar to above. We may only need the import if we're actually using BERT.

Reply:
Same reason as above